Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/core/DccTicketingCoreModule.kt

```diff
@@ -31,7 +57,7 @@ class DccTicketingCoreModule {
 }
 
 // Dummy base url to satisfy Retrofit ¯\_(ツ)_/¯
-private const val BASE_URL = "http://localhost.de"
+private const val BASE_URL = "https://localhost.de"
```

Comment:
hm... does it matter? :)

Reply:
Yes, it is causing a failure in internal builds